### ISSUE TITLE
Fatal error

### ISSUE THREAD

**Issue by serl** [pk-fr/yakpro-po#61]
Hello,
with the latest version:

```
Info:	Using [/app/yakpro-po.cnf] Config File...
Info:	yakpro-po version = 2.0.7
Info:	Process Mode		= directory
Info:	source_directory	= [api]
Info:	target_directory	= [api-obfuscated]
Obfuscating api/src/cut/Service.php
Obfuscating api/src/cut/Factory.php
Obfuscating api/src/cut/Proxy.php

Fatal error: Uncaught TypeError: Argument 1 passed to MyNodeVisitor::get_identifier_name() must implement interface PhpParser\Node, null given, called in /usr/local/yakpro-po/include/classes/parser_extensions/my_node_visitor.php on line 537 and defined in /usr/local/yakpro-po/include/classes/parser_extensions/my_node_visitor.php:45
Stack trace:
#0 /usr/local/yakpro-po/include/classes/parser_extensions/my_node_visitor.php(537): MyNodeVisitor->get_identifier_name(NULL)
#1 /usr/local/yakpro-po/PHP-Parser/lib/PhpParser/NodeTraverser.php(230): MyNodeVisitor->leaveNode(Object(PhpParser\Node\Stmt\UseUse))
#2 /usr/local/yakpro-po/PHP-Parser/lib/PhpParser/NodeTraverser.php(114): PhpParser\NodeTraverser->traverseArray(Array)
#3 /usr/local/yakpro-po/PHP-Parser/lib/PhpParser/NodeTraverser.php(223): PhpParser\NodeTraverser->traverseNode(Object(PhpParser\Node\Stmt\Use_))
#4 /usr/local/yakpro-po/PHP-Parser/lib/PhpParser/NodeTraverser.php(114): PhpParser\NodeTraverser->traverseArray(Array)
#5 /usr/local/yakpro-po/PHP-Parser/lib/PhpParser in /usr/local/yakpro-po/include/classes/parser_extensions/my_node_visitor.php on line 45
```

**Comment by pk-fr** [comment]
can you provide me an example of a small piece of source code that produces the bug?

**Comment by serl** [comment]
```php
<?php
use \Mynamespace\ClassName;
```

**Comment by pk-fr** [comment]
thanx, bug fixed wit version 2.0.8
